nixos/modules/services/databases/postgresql.nix

```diff
@@ -105,6 +105,52 @@ in
         '';
       };
 
+      ensureDatabases = mkOption {
+        default = [];
+        description = ''
+          Ensures that the specified databases exist.
+          This option will never delete existing databases, especially not when the value of this
+          option is changed. This means that databases created once through this option or
+          otherwise have to be removed manually.
+        '';
+        example = [
+          "nextcloud"
+          "matomo"
```

Comment:
Matomo does not support PostreSQL, something else should be listed as an example here.

Reply:
Thanks for letting me know!